test/sanitizer_suppressions/tsan

```diff
@@ -58,6 +58,10 @@ race:InterruptRPC
 race:src/qt/test/*
 deadlock:src/qt/test/*
 
+# Race in src/test/main.cpp
```

Comment:
21597: should also drop 
```
# race (TODO fix)
race:validation_chainstatemanager_tests
```
above

Reply:
Why?

Reply:
Hmm... well; I didn't do anything; git did it :D

Reply:
resolved in force push